### ISSUE TITLE
Fail to compile sounds example

### ISSUE THREAD

**Issue by interstar** [carp-lang/Carp#1199]
I'm new to Carp, just trying to compile the sounds example. 

And I get this error message :

```
> (load "examples/sounds.carp")
I couldn’t find the symbol 'from-cstr' at line 36, column 44 in '/media/phil/151848002EC71722/new_dev_tree/testing_frameworks/Carp/examples/sounds.carp'.

Maybe you forgot to define it?

```
Tried qualifying the name from-cstr as String.from-cstr and according to https://carp-lang.github.io/carp-docs/core/String.html#from-cstr it should be a function within the String name space. But no luck.

What's going wrong here? Is it a bug in the sounds.carp example file? Or am I doing something else wrong?

**Comment by hellerve** [comment]
The example compiles and works for me on the current master. To help debug this, may I ask for the following:
- What’s `CARP_DIR` set to? (`echo $CARP_DIR` should do the trick)
- Are you running this from the Carp main directory?
- Do other examples work, for instance `examples/vector.carp` to pick a simple one?

**Comment by hellerve** [comment]
Welcome to Carp, by the  way!

**Comment by interstar** [comment]
Thanks.

CARP_DIR doesn't seem to be set to anything, to be honest.

Which I guess may be the problem :-)

I'm running it from a directory whose listing looks like this : 

bin  core  docs  example.carp  examples  img  out  README.md  src

I guess it's the main Carp directory. (I don't quite remember how I got this, I installed it several months ago)

I run carp by typing ./bin/carp

The vector example compiles and runs fine. As does the "snake game" reptile and various others.
